fuxi-reward-claims/sig-files/jim380-64AEE4167CA1C855.txt

```diff
@@ -0,0 +1,17 @@
+-----BEGIN PGP SIGNED MESSAGE-----
+Hash: SHA512
+
+iaa1tkg8cgwlc0jt24furp26eqt8r58lsefyfz8npk
+-----BEGIN PGP SIGNATURE-----
```

Comment:
Your reward claiming file should be a signature on the delegator_address in your gentx file:

"delegator_address":"iaa1kfhee2nqrg64krqa97q3ufw9d0phzp3j83mhg4"